pkgs/applications/window-managers/awesome/default.nix

```diff
@@ -36,7 +40,8 @@ with luaPackages; stdenv.mkDerivation rec {
                   xorg.libXau xorg.libXdmcp xorg.libxcb xorg.libxshmfence
                   xorg.xcbutil xorg.xcbutilimage xorg.xcbutilkeysyms
                   xorg.xcbutilrenderutil xorg.xcbutilwm libxkbcommon
-                  xcbutilxrm ];
+                  xcbutilxrm ]
+                  ++ stdenv.lib.optionals gtk3Support [ gtk3 atk ];
```

Comment:
`atk` is propagated by `gtk3`, so it should not be necessary to include it here.

Reply:
Thanks for the info, I will try and update this tonight.

Reply:
Solved.